test/distributed/org/apache/cassandra/distributed/impl/Instance.java

```diff
@@ -590,6 +591,7 @@ public void startup(ICluster cluster)
                 DistributedTestSnitch.assign(config.networkTopology());
 
                 DatabaseDescriptor.daemonInitialization();
+                ThreadAwareSecurityManager.install();
```

Comment:
this will call LoggingSupport stuff and will check the number of virtual appenders.